WordPress/Classes/Utility/Universal Links/Routes+Notifications.swift

```diff
@@ -8,7 +8,7 @@ struct NotificationsRoute: Route {
 
 struct NotificationsNavigationAction: NavigationAction {
     func perform(_ values: [String: String], source: UIViewController? = nil, router: LinkRouter) {
-        WPTabBarController.sharedInstance().showNotificationsTab()
-        WPTabBarController.sharedInstance().popNotificationsTabToRoot()
+        RootViewCoordinator.sharedPresenter.showNotificationsTab()
+        RootViewCoordinator.sharedPresenter.popNotificationsTabToRoot()
```

Comment:
Does phase four include disabling route handlers that lead to Jetpack-powered features? With the feature flag enabled, triggering a universal link for a Jetpack feature leads to `MySitesCoordinator.fallbackBehavior()`.